pkg/ccl/backupccl/testdata/backup-restore/restore-tenants

```diff
@@ -15,15 +15,15 @@ SELECT crdb_internal.create_tenant(6);
 
 # Drop one of them.
 exec-sql
-DROP TENANT [5]
+ALTER TENANT [5] STOP SERVICE; DROP TENANT [5]
```

Comment:
We could update this test file to show that servce_state is properly restored during a tenant restore.